docs/organizations/viewing-memberships.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: View current organization memberships
+description: Learn how to view organization memberships for the currently signed-in user.
+---
+
+# View current organization memberships
+
+Clerk provides the ability for the currently signed-in user to view all the organizations they are members of.
+
+In React and Next.js applications, the [`useOrganizationList()`](/docs/references/react/use-organization-list) hook can be used to return the list of invitations or memberships for the currently active user. In JavaScript applications, the [`getOrganizationMemberships()`](/docs/references/javascript/clerk/organization-methods#get-organization-memberships) method is used to retrieve the list of organizations the current user is a part of and can be accessed through the [`Clerk` object](/docs/references/javascript/clerk/clerk).
+
+{/* Instead of viewing the current users memberships, you can also view all of the memberships for an organization.
```

Comment:
❓ Why is this paragraph commented out?

Reply:
because its not in the original docs, its a placeholder for information we need to add. I can update the comment to say TODO: etc etc

Reply:
makes sense, thanks 👍 good plan!